parachains/runtimes/collectives/collectives-polkadot/src/fellowship/mod.rs

```diff
@@ -182,8 +190,18 @@ pub type FellowshipSalaryInstance = pallet_salary::Instance1;
 
 use xcm::prelude::*;
 
+#[cfg(not(feature = "runtime-benchmarks"))]
 parameter_types! {
 	pub AssetHub: MultiLocation = (Parent, Parachain(1000)).into();
+}
+
+#[cfg(feature = "runtime-benchmarks")]
```

Comment:
if benchmark uses XcmRouter at the end, then in this case it would add different weights for `ParentAsUmp` and not for `XcmpQueue` as expected for `AssetHub` 

there is feature to open HRMP channel for benchmarks `open_outbound_hrmp_channel_for_benchmarks` e.g. here: https://github.com/paritytech/cumulus/blob/master/parachains/runtimes/bridge-hubs/bridge-hub-rococo/src/lib.rs#L1054

I dont know which benchmark/pallet here needs this, but probably the clean way, would be to extend pallet's Config for benchmark and benchmark itself with some `pre_initialization`, e.g.: here https://github.com/paritytech/substrate/blob/master/frame/system/benchmarking/src/lib.rs#L36-L45

Reply:
please check now